gapic/src/main/com/google/gapid/perfetto/views/TrackContainer.java

```diff
@@ -52,20 +52,40 @@
   private TrackContainer() {
   }
 
+  public static enum Truncate {
+    left {
```

Comment:
Left

Reply:
I had it as Left, but the eclipse language suggestion warned that usually enum constants should start with lowercase. So I reverted back to all lowercase. I'll change it to Left if needed. What do you think?

Reply:
I did a brief check in our codebase, I think `Left` is better